### PR TITLE
Configure Peakaboo (ddp) plugin

### PR DESCRIPTION
**Pull request by rjg21**
Adds the plugin python file and updates ddp settings from group_vars

Agent side of: uisautomation/lecture-capture-backend#19

### CODE REVIEW

File: roles/galicaster-capture-agent-common/files/ddp.py [hunk 1]
```diff
@@ -0,0 +1,426 @@
+# DDP galicaster plugin
```
Comment (msb) [inline roles/galicaster-capture-agent-common/files/ddp.py]: can you document where this came from

File: roles/capture-agent-common/tasks/set_facts.yml [hunk 2]
```diff
@@ -131,17 +131,6 @@
     GC_line_in_source: "{{ line_in_source.stdout }}"
   when: line_in_source.stdout != '' and inventory_hostname in groups['capture-agents-v4l2-split']
 
-- name: "If Staging, Set Peakaboo Server"
-  set_fact:
-    peakaboo_meteor: "{{ peakaboo_staging_meteor }}"
-    peakaboo_http: "{{ peakaboo_staging_http }}"
-  when: deploy_environment is defined and deploy_environment == 'staging'
```
Comment (msb) [inline roles/capture-agent-common/tasks/set_facts.yml]: is this `when` required?

Comment (rjg21) [reply roles/capture-agent-common/tasks/set_facts.yml]: Umm.. that chunk is all being removed as unnecessary (we don't use the staging option)

Comment (msb) [reply roles/capture-agent-common/tasks/set_facts.yml]: doh!

**Pull request merged**